### PR TITLE
Some additional hydra suggestions

### PR DESCRIPTION
**Pull request by JackUrb**
# Overview
Just a few general fixes following Omry's advice in #246 
- Renaming the path for structured configs from `mephisto.<abstraction>` to `mephisto/<abstraction>` to fit norms
- fixing multiline yaml by using `>`
- Updating all mephisto configs to directly use `mephisto.provider.requester_name` instead of instantiating a requester config for just that.
- Some additional string fixes that were breaking parsing (oops).

### CODE REVIEW

File: examples/simple_static_task/static_test_script.py [hunk 1]
```diff
@@ -19,9 +19,9 @@
 TASK_DIRECTORY = os.path.join(get_root_dir(), "examples/simple_static_task")
 
 defaults = [
-    {"mephisto.blueprint": BLUEPRINT_TYPE},
-    {"mephisto.architect": 'local'},
-    {"mephisto.provider": 'mock'},
+    {"mephisto/blueprint": BLUEPRINT_TYPE},
+    {"mephisto/architect": 'local'},
+    {"mephisto/provider": 'mock'},
```
Comment (omry) [inline examples/simple_static_task/static_test_script.py:24]: If you have any corresponding yaml files, you should update their directory structure as well.

File: docs/hydra_migration.md [hunk 2]
```diff
@@ -241,7 +241,7 @@ register_script_config(name='scriptconfig', module=TestScriptConfig)
...
 
 #### defaults
-It's important to note the step of creating the `defaults` entry for your `RunScriptConfig`. This will provide Hydra with some default values for keys in the created `yaml` file. The `{"mephisto.blueprint": BLUEPRINT_TYPE},` entry ensures that Hydra loads up the blueprint argument configuration that corresponds with the blueprint you're running, for instance. 
+It's important to note the step of creating the `defaults` entry for your `RunScriptConfig`. This will provide Hydra with some default values for keys in the created `yaml` file. The `{"mephisto/blueprint": BLUEPRINT_TYPE},` entry ensures that Hydra loads up the blueprint argument configuration that corresponds with the blueprint you're running, for instance. 
 
 Setting `architect` to `local` and `provider` to `mock` will make the script default to that configuration when given no arguments, however you can provide different values either in configuration files or on the command line (with `mephisto.provider.requester_name=some_requester`, or `mephisto.architect=heroku` for instance).
```
Comment (pringshia) [inline docs/hydra_migration.md:247]: does that `/` convention also apply to CLI args or nah?

**Review by pringshia — APPROVED** [review]

File: docs/hydra_migration.md (hunk 2 above)
Comment (JackUrb) [reply docs/hydra_migration.md:247]: Doesn't appear to

File: examples/simple_static_task/static_test_script.py (hunk 1 above)
Comment (JackUrb) [reply examples/simple_static_task/static_test_script.py:24]: All of these are structured configs only.

File: docs/hydra_migration.md (hunk 2 above)
Comment (omry) [reply docs/hydra_migration.md:247]: `/` applies only to config groups.
config nodes separator is `.`. in many the config groups are lining up with the config node names, but not always.

**Pull request merged**